src/main/webapp/user.js

```diff
@@ -5,7 +5,10 @@
  */
 function getUser(user) { //eslint-disable-line
   console.log('entering get user function/n');
-  fetch('/user?user=' + user)
+  const params = new URLSearchParams();
+  params.append('user', user);
+
+  fetch('/user', {method: 'GET', body: params})
```

Comment:
Can we add a TODO here to switch over to the [fetch error handler](https://github.com/googleinterns/step22-2020/blob/master/src/main/webapp/js/fetch_handler.js)?

Reply:
sure thing!

Reply:
```suggestion
  //TODO: switch to standard fetch error handler
  fetch('/user', {method: 'GET', body: params}) 
```